roulette.py

```diff
@@ -1,92 +0,0 @@
-import random
-
-def rouletteNombre (betUser,money): #boucle pour répétition si user sort de l'intervalle [1,36]   
-    choiceUser=eval(input("Sur quel numéro voulez-vous poser votre jeton de 1 à 36 :"))
-    while choiceUser <1 or choiceUser >36: 
-        choiceUser=eval(input("Sur quel numéro voulez-vous poser votre jeton de 1 à 36 :"))
-    result=random.randrange(0,36)
-    print("La bille est tombée sur le", result)
-    if result==0:
-        print("la bille est tombée sur le 0, vous avez perdu")
-        return money
-    if choiceUser==result:
-            betUser=betUser*36
-            money=money+betUser
-            print("vous avez gagné",betUser,"€")
-            return money
-    else:
-            print("vous avez perdu", betUser, "€")
-            return money
-
-def rouletteStart(): #demande la mise
-    print("faites vos jeux !")
-    betUser=eval(input("Entrer la mise en € : "))
-    return betUser
-
-def roulettePair(betUser,money): #jeu pair/impair
-    PI=["Pair","impair"]
-    choiceUser=eval(input("Précisez votre choix : Pair=0, Impair=1 : "))
-    result=random.randrange(0,36)
-    print("la bille est tombée sur le :",result)
-    if result==0:
-        print("la bille est tombée sur le 0, vous avez perdu")
-        return money
-    result=result%2
-    print("Le résultat est",PI[result])
-    if result==choiceUser:
-        betUser=betUser*2
-        money=money+betUser
-        print("vous avez gagné", betUser,"€")
-        return money
-    else:
-        print("Vous avez perdu", betUser,"€")
-        return money
-
-def rouletteCouleur(betUser,money): #Jeu couleur
-    couleur=["vert","rouge","noir","rouge","noir","rouge","noir","rouge","noir","rouge","noir","noir","rouge","noir","rouge","noir","rouge","noir","rouge","rouge","noir","rouge","noir","rouge","noir","rouge","noir","rouge","noir","noir","rouge","noir","rouge","noir","rouge","noir","rouge"]
```

Comment:
As a data scientist, you'll be using numpy pretty often.
This is a very good use case for numpy.
You can install the numpy library on your machine, and then import it at the top of your file `import numpy`
Then you can use `result = numpy.random.choice(["vert", "rouge", "noir"], p=[1/37, 18/37, 18/37])`
This will put "vert", "rouge" or "noir" directly inside result